### PR TITLE
Feature/sprint81123/feedback from sprint241023

### PR DESCRIPTION
**Pull request by bulgarchik**

### CODE REVIEW

File: FastBank.ConsoleApplication/MenuOptions.cs [hunk 1]
```diff
@@ -58,6 +61,11 @@ static public void ShowMainMenu()
                     RenderMenuByRole();
                 }
             }
+
+            Console.Clear();
+            Console.WriteLine("You successfully exited the system! Pres any key to close programm...");
```
Comment (AngelovGrowth) [inline FastBank.ConsoleApplication/MenuOptions.cs:66]: Pres -> Press
Pay attention and read everything before commit.

Comment (AngelovGrowth) [inline FastBank.ConsoleApplication/MenuOptions.cs:66]: programm -> program

File: FastBank.Services/MenuService/MenuService.cs [hunk 2]
```diff
@@ -112,5 +115,11 @@ public string InputEmail(string inquiryMsg = "Please input you email:", string e
             }
             return email ?? string.Empty;
         }
+
+        public void OperationCompleteScreen()
+        {
+            Console.WriteLine("\nOperation complete successful! Press any key to continue...");
```
Comment (AngelovGrowth) [inline FastBank.Services/MenuService/MenuService.cs:121]: Operation complete successful -> Operation completed successfully

File: FastBank.Services/MessageService/MessageService.cs [hunk 3]
```diff
@@ -198,14 +228,16 @@ public void ShowMessageMenu(User user, Message message)
...
+                                _menuService.OperationCompleteScreen();
                             }
                         }
                         break;
                     }
             }
         }
 
-        public void ShowMessages(List<Message?> messages)
+        public void ShowMessages(List<Message?> messages, User user, bool heirarchy)
```
Comment (AngelovGrowth) [inline FastBank.Services/MessageService/MessageService.cs:240]: heirarchy -> hierarchy
Edit everywhere

Comment (bulgarchik) [reply FastBank.Services/MessageService/MessageService.cs:240]: fixed

File: FastBank.Services/MenuService/MenuService.cs (hunk 2 above)
Comment (bulgarchik) [reply FastBank.Services/MenuService/MenuService.cs:121]: fixed

File: FastBank.ConsoleApplication/MenuOptions.cs (hunk 1 above)
Comment (bulgarchik) [reply FastBank.ConsoleApplication/MenuOptions.cs:66]: fixed

Comment (bulgarchik) [reply FastBank.ConsoleApplication/MenuOptions.cs]: done

**Pull request merged**